packages/core/content-manager/server/services/entity-manager.js

```diff
@@ -62,40 +62,40 @@ module.exports = ({ strapi }) => ({
     return assoc(`${CREATED_BY_ATTRIBUTE}.roles`, roles, entity);
   },
 
-  find(opts, uid, populate) {
```

Comment:
I'm wondering. If we remove the populate option, then why not also removing functions that are not anymore used?
Would it be considered as a Breaking Change even though the Admin API is not documented and not supposed to be used?
poke @alexandrebodin

Reply:
Not a breaking change to remove non public code no. But that function is still used in the single type controller ?

Reply:
how this populate was used before? You could pass a populate object in the Content API?

Reply:
yes but it was never used I believe

Reply:
Yes I checked, no use anywhere (at least internally, don't know if some users have modified Strapi's code to use it, but I don't think so)